internal/selfmonitor/prober/log_pipeline_prober_test.go

```diff
@@ -61,6 +61,25 @@ func TestLogPipelineProber(t *testing.T) {
 				},
 			},
 		},
+		{
+			name:         "pending alert",
+			pipelineName: "cls",
+			alerts: promv1.AlertsResult{
+				Alerts: []promv1.Alert{
+					{
+						Labels: model.LabelSet{
+							"alertname": "LogAgentBufferFull",
```

Comment:
What is this test case testing? Are we using the pending state on the LogAgentBufferFull as well?

Reply:
Shouldn't this test target the "NoLogsDelivered" pending state?

Reply:
No, this is the whole idea of this change. If an alert is pending, we should not map to the condition reason, only if it's firing.

Reply:
Ok, but here you are checking that if it's pending, `Healthy: true` right?